### PR TITLE
Update Nattable

### PR DESCRIPTION
**Pull request by TruongQuangSB**
- Update nattable to v2.2.0
- Update nebula to v3.0.0

### CODE REVIEW

File: java/bundles/org.eclipse.set.utils.table/src/org/eclipse/set/utils/table/sorting/AbstractSortByColumnTables.java [hunk 1]
```diff
@@ -117,13 +100,6 @@ protected NatTable createTable(final Composite parent,
 				columnHeaderLayer, new TableSortModel(bodyDataProvider), true);
 
 		final ConfigRegistry configRegistry = new ConfigRegistry();
-		configRegistry.registerConfigAttribute(
-				CellConfigAttributes.CELL_PAINTER,
-				new PaddingDecorator(
-						new FilterRowPainter(new FilterIconPainter(
-								GUIHelper.getImage("remove-filter"))), //$NON-NLS-1$
-						0, 0, 0, 5),
-				DisplayMode.NORMAL, GridRegion.FILTER_ROW);
```
Comment (TruongQuangSB) [inline java/bundles/org.eclipse.set.utils.table/src/org/eclipse/set/utils/table/sorting/AbstractSortByColumnTables.java:126]: remove-icon by filter cell in current version of nattable is default

File: java/bundles/org.eclipse.set.utils.table/src/org/eclipse/set/utils/table/sorting/AbstractSortByColumnTables.java [hunk 2]
```diff
@@ -152,10 +128,7 @@ protected NatTable createTable(final Composite parent,
 				false);
 		GridDataFactory.fillDefaults().grab(true, true).applyTo(natTable);
 		natTable.setConfigRegistry(configRegistry);
-		natTable.addConfiguration(new DefaultNatTableStyleConfiguration());
 		natTable.addConfiguration(new SingleClickSortConfiguration());
-		natTable.addConfiguration(new FilterRowCustomConfiguration());
```
Comment (TruongQuangSB) [inline java/bundles/org.eclipse.set.utils.table/src/org/eclipse/set/utils/table/sorting/AbstractSortByColumnTables.java:157]: not necessary to add default style and filterrow here. We have already use our custom theme in line 140

File: java/bundles/org.eclipse.set.utils.table/src/org/eclipse/set/utils/table/sorting/AbstractSortByColumnTables.java [hunk 3]
```diff
@@ -90,13 +80,6 @@ public void applyFilter(
 
 	}
 
-	class FilterRowCustomConfiguration extends DefaultFilterRowConfiguration {
-		public FilterRowCustomConfiguration() {
-			this.cellPainter = new FilterRowPainter(
-					new FilterIconPainter(GUIHelper.getImage("filter"))); //$NON-NLS-1$
-		}
-	}
```
Comment (TruongQuangSB) [inline java/bundles/org.eclipse.set.utils.table/src/org/eclipse/set/utils/table/sorting/AbstractSortByColumnTables.java:98]: was moved to our custom theme define

File: java/bundles/org.eclipse.set.nattable.utils/src/org/eclipse/set/nattable/utils/PlanProTableThemeConfiguration.java [hunk 4]
```diff
@@ -173,6 +163,10 @@ public PlanProTableThemeConfiguration(final NatTable natTable,
 		this.bodyLayer = bodyLayer;
 		this.bodyDataProvider = bodyDataProvider;
 
+		// IMPROVE: currently exists bug by resize row height in Nattable, when
+		// vertical alignment !== TOP
+		// See: https://bugs.eclipse.org/bugs/show_bug.cgi?id=582373
+		this.defaultVAlign = VerticalAlignmentEnum.TOP;
```
Comment (TruongQuangSB) [inline java/bundles/org.eclipse.set.nattable.utils/src/org/eclipse/set/nattable/utils/PlanProTableThemeConfiguration.java:169]: when the bug fixed, this line should remove

File: java/bundles/org.eclipse.set.nattable.utils/src/org/eclipse/set/nattable/utils/PlanProTableThemeConfiguration.java [hunk 5]
```diff
@@ -138,25 +144,9 @@ public PlanProTableThemeConfiguration(final NatTable natTable,
...
-				SetImages.WARNING_YELLOW);
-		final Image rightWarningImage = createImage(resourceManager,
-				SetImages.WARNING_RED);
-		final Image blackWarningImage = createImage(resourceManager,
-				SetImages.IC_WARNING_BLACK_18DP_1X);
-
-		this.defaultCellPainter = new PlanProTableCellPainter(
-				new PlanProRichTextCellPainter(wraptext, calculateByTextLength,
-						calculateByTextHeight),
-				leftWarningImage, rightWarningImage, blackWarningImage);
```
Comment (TruongQuangSB) [inline java/bundles/org.eclipse.set.nattable.utils/src/org/eclipse/set/nattable/utils/PlanProTableThemeConfiguration.java:158]: moved to createPainterInstance() function

**Review by PetersNils — APPROVED** [review]

**Pull request merged**